app/views/restaurants/search.html.erb

```diff
@@ -0,0 +1,8 @@
+
+
+<div class="container mt-3">
```

Comment:
@pumpkin047 how does this front end part work with our restaurant list? I think you need to get the front end code move to this file